packages/@react-aria/autocomplete/docs/useSearchAutocomplete.mdx

```diff
@@ -243,7 +243,7 @@ function Popover(props) {
           marginTop: 4
         }}>
         {children}
-        <DismissButton onDismiss={onClose} />
+        <DismissButton onDismiss={onClose} style={{transform: ''}} />
```

Comment:
We should remove these doc changes for now since they will get pushed to prod immediately but the actual code change to DismissButton won't be until we do a release. We'll need a followup PR to go in with the docs specific updates after these changes are released